etc/profile.d/xdg-local.sh

```diff
@@ -0,0 +1 @@
+export XDG_DATA_DIRS="$XDG_DATA_DIRS:$HOME/.local/share"
```

Comment:
This also looks like it should be part of the toolbox container?

Reply:
So, is the point of this to have `$HOME/.local/share` appended to the `$XDG_DATA_DIRS` environment variable, while inside the (user) toolbox? If yes, what's the reason for that?

I'm not saying there is none, but if I look at my host system (so outside of any toolbox), I see this:
```
dario@Wayrath:~> echo $XDG_DATA_DIRS
/home/dario/.local/share/flatpak/exports/share:/var/lib/flatpak/exports/share:/usr/local/share:/usr/share
```
I.e., I don't have `/home/dario/.local/share` there. Of course, this is reflected inside the toolbox since we export the `$XDG_DATA_DIRS` variable.

If there's the need for having `$HOME/.local/share` as part of the paths defined in `$XDG_DATA_DIRS`, I think we can find a way to do that, either with or without the help of a script like this. But please, can you tell us more about what we'd need it for?

Reply:
On my machine running KDE, ~/.local/share was not part of XDG_DATA_DIRS, so I assumed (possibly incorrectly) that this was needed to get the DE to see the desktop files and icons on a base MicroOS install. To be clear, it's needed outside of the container and irrelevant within the container.